src/audio/crossover/crossover.c

```diff
@@ -837,8 +837,6 @@ static int crossover_reset(struct comp_dev *dev)
 
 	cd->crossover_process = NULL;
 	cd->crossover_split = NULL;
-	cd->config = NULL;
-	cd->config_new = NULL;
```

Comment:
This was wrong in more ways than one, wasn't it? Assigning configs to NULL without freeing them first leaks memory... but that is fixed here too.